### PR TITLE
Adapts to Strapi v4

### PR DESCRIPTION
**Pull request by apriljunge**
Due to an update to Strapi v4 the API has changed.
This commit adapts the current code to work with the new graphql schema.

I tried to use variables (start_gte) like before, but I was unable to get it working

### CODE REVIEW

**Comment by derhuerst** [comment]
> I tried to use variables (start_gte) like before, but I was unable to get it working

As In, a variable cannot be used in the new `filters.start.gte` field?

File: lib/events.js [hunk 1]
```diff
@@ -92,46 +95,72 @@ const fetchEvents = async (city = null) => {
...
-						__typename
-						email
-					}
-				}
+		pagination: {
+			limit: 500 
+		}
+		filters: {
+			start: {
+				gte: "${new Date(Date.now() - 10 * DAY).toISOString()}"
```
Comment (derhuerst) [inline lib/events.js:106]: Let's move this into a variable.

File: lib/events.js [hunk 2]
```diff
@@ -149,14 +178,17 @@ query recentActions ($start_gte: String!) {
...
+			}
 		},
 	} = await res.json()
 
 	const events = []
-	for (const action of actions) {
+	for (const action of actionsData) {
 		// todo: also filter by action.group.city.name?
-		if (city !== null && slug(city) !== slug(action.location)) {
+		console.log(action.attributes.location)
```
Comment (derhuerst) [inline lib/events.js:190]: I guess you put this for debugging?

**Comment by apriljunge** [comment]
Sorry I messed this up.
I'll have to open another PR. I fixed both of your comments.

**Pull request closed without merging**